src/trade-params/TradeParams.js

```diff
@@ -85,6 +85,10 @@ export default class TradeParams extends Component {
         this.onStopTypeChange = ::this.onStopTypeChange;
         this.onStopProfitChange = ::this.onStopProfitChange;
         this.onPurchase = ::this.onPurchase;
+
+        this.state = {
+            dynamicKey: 0,
+        };
```

Comment:
Alright merging it for now.
Don't over test. I also think we can have something better than time and date picker.
Even something like a smart input like 'in 20 mins' or it might be too smart in a bad way :)
Just will need some time to think it through without the pressure.